test/emulation_helper.h

```diff
@@ -426,6 +426,7 @@ std::array<Result, VectorWidth * LocalSize * NumGroups> runEmulation(std::string
         throw vc4c::CompilationError(vc4c::CompilationStep::GENERAL, "Kernel execution failed");
 
     std::array<Result, VectorWidth * LocalSize * NumGroups> output;
+    output.fill(0);
```

Comment:
I think initializing the whole array on creation is better than filling it afterwards, e.g. replace the above line with:

    std::array<Result, VectorWidth * LocalSize * NumGroups> output{0};